docs/GUI.md

```diff
@@ -0,0 +1,32 @@
+## Schema-driven GUI
+
+Generating a GUI is done by passing data to a libary called JSON Editor. JSON supports formats for schema of type string.
+
+Example of a password object.
+```javascript
+{
+  "type": "object",
+  "properties": {
+    "password": {
+      "type": "string",
+      "format": "password"
+    }
+  }
+}
+```
+
+#### Note
```

Comment:
Combine lines 18 and 19.

**Note:** If property enum is specified, type format is ignored.